src/researches/russian/transitionWords.js

```diff
@@ -0,0 +1,91 @@
+	"благодаря тому что", "благодаря тому, что", "будто бы", "будь то", "в отношении того что", "в отношении того, что",
+	"в противовес тому что", "в противовес тому, что", "в противоположность тому что", "в противоположность тому, что",
+	"в результате", "в связи с тем что", "в связи с тем, что", "в силу того что", "в силу того, что", "в силу чего",
+	"в случае если", "в случае, если", "в сравнении с тем что", "в сравнении с тем, что", "в таком случае",
+	"в то время как", "в то время, как", "в том случае если", "в том случае, если",
+	"ввиду того что", "ввиду того, что", "вместо того чтобы", "вместо того, чтобы", "вопреки тому что",
+	"вопреки тому, что", "вплоть до того что", "вплоть до того, что", "вроде того как", "вроде того что",
+	"вроде того, как", "вроде того, что", "вследствие того что", "вследствие того, что", "вследствие чего",
+	"да еще", "да и то", "даром что", "для того чтобы", "для того, чтобы", "до тех пор пока не", "до тех пор, пока",
+	"до того как", "до того, как", "едва лишь", "едва только", "ежели бы", "за счет того что", "за счет того, что",
+	"из-за того что", "из-за того, что", "исходя из того что", "исходя из того, что", "к тому же", "как будто",
+	"как бы", "как вдруг", "как если бы", "как например", "как скоро", "как словно", "как только", "как, например",
+	"коль скоро ", "коль уж", "лишь бы", "лишь только", "между тем как", "на основании того что",
+	"на основании того, что", "на случай если", "на случай, если", "наряду с тем что", "наряду с тем, что",
+	"насчет того что", "насчет того, что", "не в пример тому как", "не в пример тому, как", "не то чтобы",
+	"невзирая на то что", "невзирая на то, что", "независимо от того что", "независимо от того, что",
+	"несмотря на то что", "несмотря на то, что", "относительно того что", "относительно того, что", "перед тем как",
+	"перед тем, как", "по мере того как", "по мере того, как", "по поводу того что", "по поводу того, что",
+	"по причине того что", "по причине того, что", "по статистике", "по сравнению с тем что", "по сравнению с тем, что",
+	"под видом того что", "под видом того, что", "под предлогом того что", "под предлогом того, что",
+	"подобно тому как", "подобно тому, как", "помимо того что", "помимо того, что", "после того как", "после того, как",
+	"потому как", "потому что", "прежде нежели", "прежде чем", "при всем том что", "при всем том, что",
+	"при условии что", "при условии, что", "прямо-таки как", "пускай бы", "равно как", "ради того чтобы", "разве что",
+	"с тем чтобы", "с тех пор как", "с той целью чтобы", "сверх того что", "смотря по тому что", "смотря по тому, что",
+	"совсем как", "судя по тому что", "судя по тому, что", "так как", "так что", "так чтобы", "тем более что",
+	"тем не менее ", "тем паче что", "то бишь", "то есть", "тогда как", "только бы", "только лишь", "только чуть",
+	"чуть лишь", "чуть только", "в частности" ];
+
+let singleIntroWordsWithoutComma = [ "бесспорно", "вишь", "во-вторых", "во-первых", "вообще-то", "впрочем", "кажись",
```

Comment:
It might actually a good idea to add a comment that specifies what we mean by intro words. So something like: `These transition words usually appear at the beginning of the sentence/clause. For the present purposes, they are treated the same way as other transition words.`